controllers/storagecluster/storageconsumer.go

```diff
@@ -103,6 +105,18 @@ func (s *storageConsumer) ensureCreated(r *StorageClusterReconciler, storageClus
 		return ctrl.Result{}, fmt.Errorf("failed to create/update storageconsumer configmap %s: %v", localStorageConsumerConfigMapName, err)
 	}
 
+	svgPre419 := &rookCephv1.CephFilesystemSubVolumeGroup{}
```

Comment:
419 can be interpreted in many ways. 
```suggestion
	svgPre41_9 := &rookCephv1.CephFilesystemSubVolumeGroup{}
```

Reply:
I suppose `svgPre4_19`

Reply:
Yes, a mistype